app/assets/javascripts/solidus_paypal_braintree/paypal_button.js

```diff
@@ -70,7 +70,16 @@ SolidusPaypalBraintree.PaypalButton.prototype._tokenizeCallback = function(token
       window.location.href = response.redirectUrl;
     },
     error: function(xhr) {
-      console.error("Error submitting transaction");
+      var errors = []
+      $.each(xhr.responseJSON.errors, function(key, values) {
+        $.each(values, function(index, value) {
+          errors.push(key + " " + value)
+        });
+      });
+      var errorText = errors.join(", ");
+
+      console.error("Error submitting transaction: " + errorText);
+      SolidusPaypalBraintree.showError(BraintreeError.DEFAULT + " " + errorText);
```

Comment:
A validation error is not "Bad", so we should not prepend all errors with the default message.